app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def new
+    @specific_post = Post.find(params[:post_id])
+    @specific_user = User.find(params[:user_id])
+  end
+
+  def create
+    @specific_post = Post.find(params[:post_id])
+    @specific_user = User.find(params[:user_id])
+    @comment = Comment.new(
+      author_id: @specific_user.id,
+      post_id: @specific_post.id,
+      text: params[:text]
+    )
```

Comment:
- [ ] Please make sure to use the `current_user` you created in the application controller and create a comment on behalf of that user as requested in the project requirements.